modules/402-ingress-nginx/templates/controller/controller.yaml

```diff
@@ -293,6 +293,8 @@ spec:
           name: uwsgi-temp-path
         - mountPath: /etc/nginx/ssl/
           name: secret-nginx-auth-tls
+        - mountPath: /tmp/nginx/
+          name: tmp-nginx
```

Comment:
Is it useful? Seems like you left something for debug here

Reply:
At least a mount point is needed to create a prometheus-nginx.socket